android/src/main/java/com/segment/analytics/kotlin/android/plugins/AndroidLifecyclePlugin.kt

```diff
@@ -47,16 +47,15 @@ class AndroidLifecyclePlugin() : Application.ActivityLifecycleCallbacks, Default
 
     override fun setup(analytics: Analytics) {
         super.setup(analytics)
-        application = analytics.configuration.application as? Application
-            ?: error("no android application context registered")
-        storage = analytics.storage
+        analytics.configuration.let {
```

Comment:
whats the difference here?

Reply:
yeah. they're the same. just want to group it together with the other assignments, since they are all visiting properties of `configuration` 😄